lib/ws-browser.js

```diff
@@ -80,14 +82,15 @@ class Transport extends EventEmitter {
 //   appProvider - client application provider
 //
 const socketFactory = (url, callback) => {
+  const cb = common.cb(callback);
```

Comment:
I'm not sure whether we need it here, it may be handled at a higher level.

/cc @lundibundi